Rise-Vision/rise-image
# Update Image component to support company logo

## Description
Retrieve logo from Branding configuration

Add dev.html demo page
Add isLogo=true image component for testing

Fix issue where file is not in list:
A file watch is registered
File is removed from the list
Watch is triggered, file is re-added to the list

## Motivation and Context
Add support for Company Branding as per the Branding epic.
Add demo page for developers
Fix issue with file updates

## How Has This Been Tested?
The changes have been tested locally via the Chrome Player with the demo page
Tested demo via Editor Preview

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
**Manual Test**  As per the above
**Automated Test** Both unit tests and integration tests were added for the change
**Monitoring** Will test the Beta release before releasing to Production. Once in Stable will monitor that no errors are occurring
**Rollback** Rollback can be done by reverting the PR and following the deployment process
**Documentation** Documentation updated as per the design doc for the Epic: https://docs.google.com/document/d/1XZUcQ6D9oWjMxrllbZKZLCDV3D8W89OyqrUMwVjEp_8/edit#
**Internal Communication** Will notify interested parties once deployed

- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
n/a
